modules/mapping-utils/src/resolveHits.js

```diff
@@ -83,14 +87,18 @@ export default type => async (
     body,
   });
 
-  let nodes = hits.hits.map(x => {
+  let edges = hits.hits.map(x => {
     let source = x._source;
     let nested_nodes = resolveNested({ node: source, nestedFields });
-    return { id: x._id, ...source, ...nested_nodes };
+
+    return {
+      searchAfter: JSON.stringify(x.sort || []),
+      node: { id: x._id, ...source, ...nested_nodes },
+    };
   });
 
   return {
-    hits: nodes,
+    edges,
```

Comment:
is this not a fairly major / breaking change?

Reply:
nope! that change along with https://github.com/overture-stack/arranger/pull/269/files#diff-d51be1b4aaffd8cba6ee072e7ff32159L53
makes the graphql response the same.

Reply:
ohh cool 👍